test/package.json

```
@@ -4,9 +4,9 @@
   "private": true,
   "scripts": {
     "test": "node --preserve-symlinks test.js test",
-    "coverage": "cross-env NODE_ENV=test nyc --reporter=lcov --reporter=text-summary npm test",
-    "coverage-ci": "cross-env NODE_ENV=test nyc --reporter=lcov npm test",
-    "coverage-report": "nyc report"
+    "coverage": "cross-env NODE_ENV=test c8 --reporter=lcov --reporter=text-summary npm test",
+    "coverage-ci": "cross-env NODE_ENV=test c8 --reporter=lcov npm test",
```

This should have already been just "lcovonly" since ci only needs the coverage report.
```suggestion
    "coverage-ci": "cross-env NODE_ENV=test c8 --reporter=lcovonly npm test",
```